ozaria/site/components/interactive/PageInteractive/draggableOrdering/OrderingSlots.vue

```diff
@@ -0,0 +1,101 @@
+<script>
+  import BaseDraggableSlot from '../BaseDraggableSlot'
+
+  export default {
+    components: {
+      'base-draggable-slot': BaseDraggableSlot
+    },
+
+    props: {
+      draggableGroup: {
+        type: String,
+        required: true
+      },
+
+      numSlots: {
+        type: Number,
+        default: undefined
+      },
+
+      value: {
+        type: Array,
+        default: () => ([])
+      }
+    },
+
+    data () {
+      let slotData = []
+
+      if (typeof this.numSlots !== 'undefined') {
+        slotData = Array(this.numSlots)
+          .fill(undefined)
+      } else {
+        slotData = this.value
+      }
+
+      return {
+        slotData
+      }
+    },
+
+    methods: {
+      changed () {
+        this.$emit(
+          'input',
+          this.slotData.map(s => s[0])
+        )
+      }
+    }
+  }
+</script>
+
+<template>
+  <div class="ordering-slots-container">
+    <base-draggable-slot
+      v-for="(slot, i) in slotData"
+      :key="i"
+
+      v-model="slotData[i]"
+
+      :draggable-group="draggableGroup"
+
+      class="draggable-slot"
+
+      @change="changed"
+    />
+  </div>
+</template>
+
+<style scoped lang="scss">
+  .ordering-slots-container {
+    width: 25%;
+
+    display: flex;
+    flex-direction: column;
+
+    align-items: center;
+    justify-content: space-evenly;
+
+    /deep/ .draggable-slot {
```

Comment:
I haven't seen this syntax before?

Reply:
This syntax is for accessing the shadow dom. It also appears to be deprecated in CSS. I am assuming that scss is transforming this into a browser compatible selector.
Reference: https://stackoverflow.com/a/25609679